modules/virt-creating-configmap.adoc

```diff
@@ -114,9 +117,11 @@ $ cat <<EOF | oc apply -f -
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: os-configmap
+  name: vm-import-controller-config
```

Comment:
That doesn't need to change.

Reply:
Does this mean we have two parameters, `metadata: name` and `data: osConfigMap.name`, for the name of the custom ConfigMap?

Reply:
> Point 6 looks good, it would only be valuable to explain that value of osConfigMap.name (2) has to correspond to the name of the config map from point 4.

I don't think that it is necessary to say that the ConfigMap name must be the same because we don't mention or suggest the option of changing the ConfigMap name.

Reply:
No, there is no `data: osConfigMap.name`in this particular config map, it is in the another, new one. User needs **two** config maps to make custom template mapping work:
- first - with the os mapping, which can be named freely by the user (point 4 of the documentation) - nothing changes here;
- second one that specifies the namespace and name of the first config map in `osConfigMap.name` and `osConfigMap.namespace`. The second map is called `vm-import-controller-config`.